### ISSUE TITLE
Logs show: WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by ovizii** [oznu/docker-guacamole#67]
**Describe Your Problem:**
Not really a problem per se, just wanted to report an error I saw in the logs

**Logs:**
docker logs guacamole 

```
27-Aug-2020 09:08:17.322 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
27-Aug-2020 09:08:17.328 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["ajp-nio-8009"]
27-Aug-2020 09:08:17.336 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [3,468] milliseconds
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.postgresql.jdbc.TimestampUtils (file:/config/guacamole/lib/postgresql-42.1.4.jar) to field java.util.TimeZone.defaultTimeZone
WARNING: Please consider reporting this to the maintainers of org.postgresql.jdbc.TimestampUtils
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

```

**Environment:**

* **Guacamole Version**: oznu/guacamole:1.2.0-amd64
* **Operating System**: Docker

**Comment by jamesmuldrow** [comment]
> **Describe Your Problem:**
> Not really a problem per se, just wanted to report an error I saw in the logs
> 
> **Logs:**
> docker logs guacamole
> 
> ```
> 27-Aug-2020 09:08:17.322 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
> 27-Aug-2020 09:08:17.328 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["ajp-nio-8009"]
> 27-Aug-2020 09:08:17.336 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [3,468] milliseconds
> WARNING: An illegal reflective access operation has occurred
> WARNING: Illegal reflective access by org.postgresql.jdbc.TimestampUtils (file:/config/guacamole/lib/postgresql-42.1.4.jar) to field java.util.TimeZone.defaultTimeZone
> WARNING: Please consider reporting this to the maintainers of org.postgresql.jdbc.TimestampUtils
> WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
> WARNING: All illegal access operations will be denied in a future release
> ```
> 
> **Environment:**
> 
> * **Guacamole Version**: oznu/guacamole:1.2.0-amd64
> * **Operating System**: Docker

Did you ever figure out what is causing this warning?

**Comment by ovizii** [comment]
nope, sorry, still seeing those in my logs

**Comment by krisarmstrong** [comment]
> nope, sorry, still seeing those in my logs

I am also seeing this in ubuntu 20.04 with guacamole 1.2

```
Oct 29 15:13:09 guacamole tomcat9[918]: WARNING: An illegal reflective access operation has occurred
Oct 29 15:13:09 guacamole tomcat9[918]: WARNING: Illegal reflective access by org.apache.ibatis.ognl.OgnlRuntime (file:/etc/guacamole/extensions/guacamole-auth-jdbc-mysql-1.2.0.jar) to method java.util.Collections$EmptySet.isEmpty()
Oct 29 15:13:09 guacamole tomcat9[918]: WARNING: Please consider reporting this to the maintainers of org.apache.ibatis.ognl.OgnlRuntime
Oct 29 15:13:09 guacamole tomcat9[918]: WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
Oct 29 15:13:09 guacamole rsyslogd: action 'action-8-builtin:omfile' resumed (module 'builtin:omfile') [v8.2001.0 try https://www.rsyslog.com/e/2359 ]
Oct 29 15:13:09 guacamole rsyslogd: action 'action-8-builtin:omfile' suspended (module 'builtin:omfile'), retry 0. There should be messages before this one giving the reason for suspension. [v8.2001.0 try https://www.rsyslog.com/e/2007 ]
Oct 29 15:13:09 guacamole rsyslogd: action 'action-8-builtin:omfile' resumed (module 'builtin:omfile') [v8.2001.0 try https://www.rsyslog.com/e/2359 ]
Oct 29 15:13:09 guacamole rsyslogd: action 'action-8-builtin:omfile' suspended (module 'builtin:omfile'), retry 0. There should be messages before this one giving the reason for suspension. [v8.2001.0 try https://www.rsyslog.com/e/2007 ]
```

**Comment by neutronscott** [comment]
I have built this for aarch64 for my amazon instance and saw these warnings. This was fixed a few years ago in a later postgres jdbc. They just released version 42.2.18. Change this and see what you get:

`# Install guacamole-client and postgres auth adapter
RUN set -x \
  && rm -rf ${CATALINA_HOME}/webapps/ROOT \
  && curl -SLo ${CATALINA_HOME}/webapps/ROOT.war "http://apache.org/dyn/closer.cgi?action=download&filename=guacamole/${GUAC_VER}/binary/guacamole-${GUAC_VER}.war" \
  && curl -SLo ${GUACAMOLE_HOME}/lib/postgresql-42.2.18.jar "https://jdbc.postgresql.org/download/postgresql-42.2.18.jar" \
  && curl -SLO "http://apache.org/dyn/closer.cgi?action=download&filename=guacamole/${GUAC_VER}/binary/guacamole-auth-jdbc-${GUAC_VER}.tar.gz" \
  && tar -xzf guacamole-auth-jdbc-${GUAC_VER}.tar.gz \
  && cp -R guacamole-auth-jdbc-${GUAC_VER}/postgresql/guacamole-auth-jdbc-postgresql-${GUAC_VER}.jar ${GUACAMOLE_HOME}/extensions/ \
  && cp -R guacamole-auth-jdbc-${GUAC_VER}/postgresql/schema ${GUACAMOLE_HOME}/ \
  && rm -rf guacamole-auth-jdbc-${GUAC_VER} guacamole-auth-jdbc-${GUAC_VER}.tar.gz`

**Comment by neutronscott** [comment]
Well there are more. Maybe it's easier to just keep using JDK 8 ha

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.ibatis.ognl.OgnlRuntime (file:/config/guacamole/extensions/guacamole-auth-jdbc-postgresql-1.2.0.jar) to method java.util.Collections$EmptySet.isEmpty()
WARNING: Please consider reporting this to the maintainers of org.apache.ibatis.ognl.OgnlRuntime
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```
